build_xcframework.sh

```diff
@@ -0,0 +1,30 @@
+#!/bin/sh
```

Comment:
I think in `main` now there is a full xcframework build script that actually uses dynamic libraries. I had had quite some issues where iOS or Android could not find the correct symbols on static libraries and this would keep the build a bit smaller.

Reply:
I was unaware that the script had been added to the repository. I think I can remove mine and use that one. I'll submit a patch.